core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -250,20 +249,50 @@ public T getDefault() {
 
     @Override
     public String toString() {
-      return name;
+      return debugString;
+    }
+
+    /**
+     * Factory method for creating instances of {@link Key}.
+     *
+     * @param debugString a string used to describe this key, used for debugging.
+     * @param defaultValue default value to return when value for key not set
+     * @param <T> Key type
+     * @return Key object
+     * @deprecated Use {@link #create} or {@link #createWithDefault} instead.
+     */
+    @Deprecated
```

Comment:
Without the experimental API, we can't delete this. It needs to be clear this API is going to be deleted.